x-pack/platform/plugins/shared/cloud/server/cloud_data/persist_token.test.ts

```diff
@@ -110,6 +145,52 @@ describe('persistTokenCloudData', () => {
     );
   });
 
+  it('updates an existing saved object if resourceData is provided and different', async () => {
```

Comment:
nit: not sure what test name was intended

```suggestion
  it('updates an existing saved object if resourceData is provided', async () => {
```